lcls-twincat-optics/lcls_twincat_optics_plc/lcls_twincat_optics_plc.plcproj

```diff
@@ -16,7 +16,7 @@
     <CombineIds>false</CombineIds>
     <Company>SLAC</Company>
     <Released>false</Released>
-    <Title>lcls-twincat-optics</Title>
+    <Title>lcls-twincat-optics-nrw</Title>
```

Comment:
I think this is the only issue I see, remember to change the project name back

Reply:
switched projected back to default. ready to merge @ZLLentz